tests/e2e/specs/modules/analytics/write-scope-requests.test.js

```diff
@@ -111,22 +114,22 @@ describe( 'Analytics write scope requests', () => {
 
 	beforeEach( async () => {
 		scope = 'https://www.googleapis.com/auth/analytics.provision';
-		bypassCreatePropertyRequest = true;
-		bypassCreateProfileRequest = true;
+		interceptCreatePropertyRequest = false;
+		interceptCreateProfileRequest = false;
 
-		await activatePlugin( 'e2e-tests-proxy-auth-plugin' );
 		await activatePlugin( 'e2e-tests-site-verification-plugin' );
 		await activatePlugin( 'e2e-tests-oauth-callback-plugin' );
+
+		await setupSiteKit();
 	} );
 
 	afterEach( async () => {
 		await deactivateUtilityPlugins();
 		await resetSiteKit();
 	} );
 
-	it( 'creating an analytics account when not having the https://www.googleapis.com/auth/analytics.provision scope yet.', async () => {
+	it( 'it prompts for additional permissions during a new Analytics account creation if the user has not granted the Analytics provisioning scope', async () => {
```

Comment:
Sorry this was probably unclear, but the "it" would still come from the function, I didn't mean to add it in the test name.
```suggestion
	it( 'prompts for additional permissions during a new Analytics account creation if the user has not granted the Analytics provisioning scope', async () => {
```